app/config/FrontendAppConfig.scala

```diff
@@ -59,7 +52,8 @@ class FrontendAppConfig @Inject() (configuration: Configuration,
     "cymraeg" -> Lang(WELSH)
   )
 
-  def routeToSwitchLanguage: String => Call =
-    (lang: String) => routes.LanguageSwitchController.switchToLanguage(lang)
+  val cacheTtlSeconds: Int = configuration.get[Int]("mongodb.timeToLiveInSeconds")
```

Comment:
should be Long instead; Type-mismatch error

Reply:
I dont see any reasons to put it as long, int is more than enough you will never ever overflow int value here so why use a type which use more space to store value?

Reply:
Though tests working fine; Type mis-match error is occuring

Reply:
I didnt have this warnings/error in my idea so this is why I was asking about the change but looking into the lib itself it is expecting a long value so it amkes sense will fix it soon

Reply:
done

Reply:
done